hassio/src/hassio-markdown-dialog.js

```diff
@@ -1,18 +1,18 @@
 import "@polymer/app-layout/app-toolbar/app-toolbar";
 import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
-import "@polymer/paper-dialog/paper-dialog";
 import "@polymer/paper-icon-button/paper-icon-button";
 import { html } from "@polymer/polymer/lib/utils/html-tag";
 import { PolymerElement } from "@polymer/polymer/polymer-element";
 
 import "../../src/components/ha-markdown";
 import "../../src/resources/ha-style";
+import "../../src/panels/lovelace/components/dialog/ha-dialog";
```

Comment:
this shouldn't be part of the lovelace components but instead be part of `src/components` and name should match the component: `ha-paper-dialog`.

Reply:
See my comment at the top. Knew you were going to say that...

Reply:
Done